semantics/language/common/expr/multiplicative.k

```diff
@@ -60,12 +60,12 @@ module C-COMMON-EXPR-MULTIPLICATIVE
                andBool (max(T) >=Int I1 /Int I2)
                andBool T ==Type T'
           [structural]
-     rule (.K => UNDEF("CEM2", "Modulus by 0.", "6.5.5:5") )
+     rule (.K => UNDEF("CEMX2", "Modulus by 0.", "6.5.5:5, J.2:1 item 45") )
           ~> L:RValue % tv(0, T':Type)
           requires isIntegerType(type(L)) andBool isPromoted(type(L))
                andBool type(L) ==Type T'
           [structural]
-     rule (.K => UNSPEC("CEM3", "Signed overflow.", "6.3.1.3:3") ) ~>
+     rule (.K => UNSPEC("CEMX3", "Signed overflow.", "6.3.1.3:3") ) ~>
```

Comment:
you should also add the bullet items (section J.1) for the unspecified behaviors.